README.md

```diff
@@ -1,3 +1,3 @@
 # hello-world
 Just another repository
-Hello these are thec hanges I am making to puul the request.
+Hello these are the changes I am making in the pull request.
```

Comment:
git checkout master
git pull
git checkout <branch>
git merge master
[ ... resolve any conflicts ... ]
git add [files that were conflicted]
git commit
git push